deployment/postgres-init-db/sql/tables/sequencing/workspace.sql

```diff
@@ -0,0 +1,31 @@
+create table sequencing.workspace (
+  id integer generated always as identity,
+
+  name text not null,
+
+  created_at timestamptz not null default now(),
+  owner text,
+  updated_at timestamptz not null default now(),
+  updated_by text,
+
+  constraint workspace_synthetic_key
+    primary key (id),
+  foreign key (owner)
+    references permissions.users
+    on update cascade
+    on delete set null,
+  foreign key (updated_by)
+    references permissions.users
+    on update cascade
+    on delete set null
+);
+
+comment on table sequencing.workspace is e''
+  'A container for multiple sequences.';
```

Comment:
Minor, nonblocking: Is there a more specific way to describe what a sequencing workspace is? If not, feel free to resolve.